docs/pages/versions/unversioned/sdk/facebook.md

```diff
@@ -26,9 +26,9 @@ Follow [Facebook's developer documentation](https://developers.facebook.com/docs
 
 Then follow these steps based on the platforms you're targetting. This will need to be done from the [Facebook developer site](https://developers.facebook.com/).
 
-The Android Play Store Expo client will use the Facebook App ID that you provide, however, all Facebook API calls in the **iOS App Store Expo client will use Expo's own Facebook App ID**. This is due to underlying configuration limitations, but the good news is it means less setup for you! The slight downside to this is that you can't customize which permissions your app requests from Facebook (like `user_photos` or `user_friends`), or integrate Facebook login with other services like Firebase auth. If you need that functionality on iOS, you have two options:
+The Android Play Store Expo Go will use the Facebook App ID that you provide, however, all Facebook API calls in the **iOS App Store Expo Go will use Expo's own Facebook App ID**. This is due to underlying configuration limitations, but the good news is it means less setup for you! The slight downside to this is that you can't customize which permissions your app requests from Facebook (like `user_photos` or `user_friends`), or integrate Facebook login with other services like Firebase auth. If you need that functionality on iOS, you have two options:
 
-- Build a [custom Expo client app](../../../guides/adhoc-builds.md)
+- Build a [custom Expo Go app](../../../guides/adhoc-builds.md)
```

Comment:
We should decide how we want to refer to these custom clients, which are close to obsolete.

Reply:
indeed